packages/elemental-theme/package.json

```diff
@@ -15,8 +15,8 @@
     "!.*"
   ],
   "scripts": {
-    "build:light": "theme-compiler light --variant=light",
-    "build:dark": "theme-compiler dark --variant=dark",
+    "build:light": "theme-compiler light --variant=light --styles=event",
```

Comment:
Maybe use `--registration` instead of `--styles`.

Reply:
Updated.